### PR TITLE
Contact us form

### PR DESCRIPTION
**Pull request by bjss-colin**

### CODE REVIEW

**Review by Okechukwu802 — DISMISSED** [review]

**Review by Okechukwu802 — DISMISSED** [review]

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/PublicBrowse/Homepage/ContactUs.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+            CommonActions.ElementShowingCorrectErrorMessage(ContactUsObjects.ContactMethodInputError, "Error: Select your reason for contacting us").Should().BeTrue();
+            CommonActions.ElementShowingCorrectErrorMessage(ContactUsObjects.FullNameInputError, "Enter a full name").Should().BeTrue();
+            CommonActions.ElementShowingCorrectErrorMessage(ContactUsObjects.MessageInputError, "Enter your message for us").Should().BeTrue();
+            CommonActions.ElementShowingCorrectErrorMessage(ContactUsObjects.EmailAddressInputError, "Enter an email address").Should().BeTrue();
+            CommonActions.ElementShowingCorrectErrorMessage(ContactUsObjects.PrivacyPolicyInputError, "Error: Confirm you have read and understood the privacy policy").Should().BeTrue();
+        }
+
+        [Fact]
+        public void ContactUs_InvalidEmailFormat_ERrorSummary()
```
Comment (steves-bjss) [inline tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/PublicBrowse/Homepage/ContactUs.cs]: Typo in error

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Validation/ContactUsModelValidator.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            RuleFor(m => m.ContactMethod)
+                .NotNull()
+                .WithMessage(ContactMethodMissingErrorMessage);
+
+            RuleFor(m => m.PrivacyPolicyAccepted)
+                .Equal(true)
+                .WithMessage(PrivacyPolicyNotAcceptedErrorMessage);
+        }
+
+        public object TestValidator { get; internal set; }
```
Comment (steves-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Validation/ContactUsModelValidator.cs]: Is this property used anywhere?

Comment (ghost) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Validation/ContactUsModelValidator.cs]: No, Intellisense does misbehave from time to time due to laptop performance. This time I missed one, removed.

File: tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/PublicBrowse/Homepage/ContactUs.cs (hunk 1 above)
Comment (ghost) [reply tests/NHSD.GPIT.BuyingCatalogue.E2E.PublicBrowseTests/Areas/PublicBrowse/Homepage/ContactUs.cs]: Resolved

**Review by steves-bjss — APPROVED** [review]

**Review by Okechukwu802 — APPROVED** [review]

**Pull request merged**